build.gradle

```diff
@@ -13,28 +13,37 @@ group = project.maven_group
 project.ext.lwjglVersion = "3.3.2"
 project.ext.jomlVersion = "1.10.4"
 project.ext.winNatives = "natives-windows"
+project.ext.winArm64Natives = "natives-windows-arm64"
 project.ext.linuxNatives = "natives-linux"
+project.ext.linuxArm64Natives = "natives-linux-arm64"
+project.ext.linuxArm32Natives = "natives-linux-arm32"
```

Comment:
Why do you need arm32?
I think that would be an overkill and arm64 and x86_64 should be enough for Windows, Linux and macOS

Reply:
answer is in the first message. 

>  the best (imho) cross platform launcher supports ARM32/ARM64 Linux

I'm referring to Prism Launcher here.

arm32 is extremely popular on (more so than arm64 unfortunately) on SBCs where a performant vulkan mod is very valuable. some of this hardware uses armv7 CPUs even (those are not 64bit cpus).